hles/transformation/src/main/scala/org/broadinstitute/monster/dap/dog/DemographicsTransformations.scala

```diff
@@ -56,10 +56,28 @@ object DemographicsTransformations {
   def mapAge(rawRecord: RawRecord, dog: HlesDogDemographics): HlesDogDemographics =
     rawRecord.getOptionalBoolean("dd_dog_birth_year_certain").fold(dog) { ageCertain =>
       if (ageCertain) {
-        val formYear = rawRecord.getRequired("dd_dog_current_year_calc").toInt
+        val dogId = rawRecord.id
+        val formYear = rawRecord.getOptional("dd_dog_current_year_calc") match {
+          case Some(year) => year.toInt
+          case None       =>
+            // we're throwing these errors instead of logging them because we want to completely skip records missing this value. they get logged
+            // by the logic up the chain that catches the error.
```

Comment:
i don't like this comment duplication but i feel it's important to note right next to both places where it's happening